packages/core/strapi/src/cli/commands/transfer/command.ts

```diff
@@ -32,9 +32,11 @@ const command = () => {
           `URL of the remote Strapi instance to send data to`
         ).argParser(parseURL)
       )
+
       .addOption(
         new Option('--to-token <token>', `Transfer token for the remote Strapi destination`)
       )
+      .addOption(new Option('-i, --info', 'log information about the export process'))
```

Comment:
Shouldn't this be `--log-level` or `verbosity` to choose the level? That way warnings could be disabled if desired, and we can add a "debug" level later that includes even more detailed information

Reply:
Bassel and I had a discussion on that. We decided to make info a boolean because we think that, either way, the warnings and errors will be displayed by default. So we are only determining if info will be displayed here or not. What do you think?

Reply:
Yeah, less so for warn/error, but I'm afraid we'll want to add 'debug' level soon that logs -everything- and we will have limited ourselves and will be forced to add another option like --debug

Reply:
yeah I have been thinking about this, in terms of changing it to `--verbose` or something nothing prevents us from changing things later if we need to, or even just adding `--debug` option, which is also pretty standard.
but adding log levels now when I know there is only one level and probably will be the case for a very long while I think is unnecessary and will only cause confusion

Reply:
in the future we could have multiple level of logs, though

so, even if it's just one level for now, probably we should already support this scenario